var/spack/repos/builtin/packages/neuron/package.py

```diff
@@ -45,6 +45,7 @@ class Neuron(CMakePackage):
     depends_on("py-pytest",   when="+python+tests")
     depends_on("readline")
     depends_on("caliper",     when="+caliper")
+    depends_on("py-numpy",    when="+coreneuron", type='run')
```

Comment:
@ikitayama : numpy is needed for neuron and not anything specific for coreneuron. So I would drop when clause.